### ISSUE TITLE
Too much ram usage for service running.

### ISSUE THREAD

**Issue by killoid** [8bitPit/Niagara-Issues#84]
i've included an ss for the issue. i really hope that those two gms service running is removed in future.
![screenshot_settings_20180907-001233](https://user-images.githubusercontent.com/6173607/45180407-1fbc8d80-b23d-11e8-9282-99fc78baed46.png)

**Comment by 8bitPit** [comment]
I’m glad to tell you, that this is not a real bug. The running services sections is “buggy”. The gms.service does not run exclusively for that app, instead it shows only up under the app that has the highest priority. If you disable notification dots for Niagara and have them enabled on nova for example, the service will appear there (I tested that with other launchers as well). It’s the same service and it will run with or without Niagara Launcher. So, don’t worry, it’s just confusingly displayed!

I found some helpful information on GitHub (shadowsocks/shadowsocks-android, Issue #1708) 
“I wouldn't worry too much about it. I did a dumpsys and MeasurementBrokerService which is held by com.google.android.gms is bound to 10 different apps. It's showing up in Shadowsocks because Shadowsocks's VPN service is a foreground service and bound to system, and therefore it has the highest priority. If you turn off Shadowsocks, you should be able to observe that com.google.android.gms now appears under some other app's memory usage (for me it's Nova Launcher). Shadowsocks VPN service itself contributes a very moderate extra memory footprint (<<5MB) for this service and that 100MB is the total memory held by that process.”

To sum it up there is no problem there and you can ignore that high number. It’s not representing the memory usage of Niagara Launcher.
Thank you, for your concerns though.

**Comment by 8bitPit** [comment]
![image](https://user-images.githubusercontent.com/9284845/45253968-c7d27380-b36f-11e8-9b12-07b709b7e10a.png)

Here is an example when nova launcher has the gms service attached. If you would force stop nova, just some other application would get the service. Of course that also goes for Niagara.

**Comment by killoid** [comment]
thank you so much for the clarification!
